eng/platform-matrix.yml

```diff
@@ -33,11 +33,6 @@ jobs:
         image: ubuntu-16.04-cross-14.04-23cacb0-20190528233931
```

Comment:
Is this image still valid? Just to confirm, this means use a 14.04 image on a 16.04 platform right?
Do we want to move to 16.04 here?

Reply:
We build arm32 coreclr targetting Ubuntu 14.04 to ensure the most widespread distro versions support. This image is running on Ubuntu 16.04 with rootfs for cross-building containing 14.04.